src/main/java/de/retest/web/selenium/RecheckWebOptions.java

```diff
@@ -0,0 +1,15 @@
+package de.retest.web.selenium;
+
+import de.retest.recheck.RecheckOptions;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@Getter
+public class RecheckWebOptions extends RecheckOptions {
```

Comment:
New public API, therefore I suggest to add some Javadoc to the class as well as the field?